src/SprykerEco/Zed/Heidelpay/Persistence/Propel/Schema/spy_heidelpay.schema.xml

```diff
@@ -119,13 +119,13 @@
     <table name="spy_payment_heidelpay_transaction_log">
         <column name="id_payment_heidelpay_transaction_log" type="INTEGER" primaryKey="true" autoIncrement="true"/>
         <column name="fk_sales_order" required="true" type="INTEGER"/>
-        <column name="id_transaction_unique" required="false" type="VARCHAR"/>
-        <column name="transaction_type" required="false" type="VARCHAR"/>
-        <column name="processing_code" required="false" type="VARCHAR"/>
-        <column name="response_code" required="false" type="VARCHAR"/>
-        <column name="redirect_url" type="VARCHAR"/>
-        <column name="request_payload" required="false" type="LONGVARCHAR"/>
-        <column name="response_payload" required="true" type="LONGVARCHAR"/>
+        <column name="id_transaction_unique" required="false" type="VARCHAR" size="255"/>
+        <column name="transaction_type" required="false" type="VARCHAR" size="255"/>
+        <column name="processing_code" required="false" type="VARCHAR" size="255"/>
+        <column name="response_code" required="false" type="VARCHAR" size="255"/>
+        <column name="redirect_url" type="VARCHAR" size="255"/>
```

Comment:
Redirect url might be much bigger than 255. You need to check all fields and set corresponding length. Even LONGVARCHAR if needed.

Reply:
Please check ALL fields. Because defaul PostgreSQL length is much bigger than 255.

Reply:
Described in the change log like |redirect_url | 5000 | The url may collect a huge amount of parameters | 
But haven't fixed.

Reply:
Also, looks that we should add size property for 
```
<column name="request_payload" required="false" type="LONGVARCHAR"/>
<column name="response_payload" required="true" type="LONGVARCHAR"/>
```

Reply:
Removed changelogs regarding size properties due to the size properties already present.

Reply:
So, we removed from this PR all size changes, because it's doesn't align with the current bug. It was here just because of CI validation issues. But It was fixed into another PR which already merged.
Regarding the potential issue with `redirect_url` size, I suggest creating a new bug ticket.